tdms/tests/matlab/README.md

```diff
@@ -0,0 +1,7 @@
+### MATLAB tests
+
+The unit tests in this directory can be run with
+
+```bash
+matlab -nodisplay -nosplash -nodesktop -r "run('run_tests.m');exit;"
```

Comment:
```suggestion
matlab -batch run_tests
```
This is how `MATLAB` recommends running a script without launching a window. I can't find the specific documentation right now I'm afraid (MATLAB documentaiton is a maze...)

Reply:
love it 👍🏼